cedar-policy/src/api.rs

```diff
@@ -1709,11 +1720,12 @@ impl Policy {
         serde_json::to_value(&self.est)
     }
 
-    /// Create a `Policy` from its AST representation only. The EST will
-    /// reflect the AST structure. When possible, don't use this method and
-    /// create the EST from the policy text or CST instead, as the conversion
-    /// to AST is lossy. ESTs generated by this method will reflect the AST and
-    /// not the original policy syntax.
+    /// Create a `Policy` from its AST representation only. The `LosslessPolicy`
+    /// will reflect the AST structure. When possible, don't use this method and
+    /// create the `Policy` from the policy text, CST, or EST instead, as the
+    /// conversion to AST is lossy. ESTs for policies generated by this method
+    /// will reflect the AST and not the original policy syntax.
+    #[cfg_attr(not(feature = "partial-eval"), allow(unused))]
```

Comment:
I think the release branch should grab this `cfg_attr` but not the docstring change above (which is from a different PR that I don't think we're including in 2.3.1)